packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+import { Notice } from '@cerner/terra-docs';
+
+# v7.0.0 Upgrade Guide
+
+This document will provide information on upgrading from `terra-dev-site` `6.x.x` to `7.0.0`.
+
+`terra-dev-site` v7 is about 70% new. The primary goals of this upgrade is to enable webpack 5, prepare for `terra-application` v2, and remove the `terra-dev-site` build intermediate step by converting to a pure webpack plugin.
```

Comment:
```suggestion
`terra-dev-site` v7 is about 70% new. The primary goals of this upgrade is to enable webpack 5, prepare for `terra-application` v2, and remove the intermediate `terra-dev-site` build step by converting to a pure webpack plugin.
```
Does this speed up build times?

Reply:
I don't have data on this but probably should. I'll gather before and afters for terra-ui for both webpack speed and bundle size.